docs/src/components/landing/DemoVideo.js

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+
+const Video = styled('video')(({ theme }) => ({
+  borderStyle: 'solid',
+  borderColor:
+    theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : theme.palette.grey[100],
+  borderRadius: '10px 10px 0 0 ',
+  overflow: 'hidden',
+  objectFit: 'cover',
+  objectPosition: 'top',
+  outline: `1px solid ${
+    theme.palette.mode === 'dark' ? theme.palette.primaryDark[400] : theme.palette.grey[300]
+  }`,
+}));
+
+export default function DemoVideo() {
+  return (
+    <React.Fragment>
+      <Video
+        sx={{
+          height: { lg: 655 },
+          borderWidth: { xs: '10px 10px 0 10px', sm: '20px 20px 0 20px' },
+        }}
+        width={1400}
```

Comment:
Do we need this width attribute?

```suggestion
```

If we do, the actual value with the largest viewport seems to be:

```suggestion
        width={1100}
```